scripts/build.sh

```diff
@@ -148,9 +148,9 @@ curl --fail -sL -o $binary_cache/ovs-common.deb http://mirrors.kernel.org/ubuntu
 curl --fail -sL -o $binary_cache/ovs-switch.deb http://mirrors.kernel.org/ubuntu/pool/main/o/openvswitch/openvswitch-switch_2.5.2-0ubuntu0.16.04.3_amd64.deb
 
 env_file=$output_dir/install/ansible/env.json
-sed -i.bak "s#.*auth_proxy_local_install.*#  \"auth_proxy_local_install\": True,#g" $env_file
-sed -i.bak "s#.*contiv_network_local_install.*#  \"contiv_network_local_install\": True,#g" $env_file
-
+sed -i.bak s#__AUTH_PROXY_LOCAL_INSTALL__#true#g $env_file
+sed -i.bak s#__CONTIV_NETWORK_LOCAL_INSTALL__#true#g $env_file
+cat $env_file
```

Comment:
remove?

Reply:
ditto

Reply:
if intentional, seems you should have a title or something to indicate what you just spewed out?

Reply:
i added `sed -x` here so it should have some hints ... sure i can add some more info here